src/js/drawer.js

```diff
@@ -88,7 +88,7 @@ class Drawer {
 	 */
 	toggleDrawer () {
 		this.nav.classList.toggle(this.class.open);
-		this.burger.classList.toggle('o-header-services__hambuger--open');
+		this.burger.classList.toggle('o-header-services__hamburger--open');
```

Comment:
```suggestion
```

Good spot. This class was actually [removed recently](https://github.com/Financial-Times/o-header-services/pull/78/files#diff-053416ca327b2589de1feeac0090e354L63), so we can remove here too.

Reply:
Removed it